openassessment/assessment/api/staff.py

```diff
@@ -56,6 +58,41 @@ def assessment_is_finished(submission_uuid, requirements):
     return True
 
 
+def on_start(submission_uuid):
+    """Create a new staff workflow for a student item and submission.
+
+    Creates a unique staff workflow for a student item, associated with a
+    submission.
+
+    Args:
+        submission_uuid (str): The submission associated with this workflow.
+
+    Returns:
+        None
+
+    Raises:
+        StaffAssessmentInternalError: Raised when there is an internal error
+            creating the Workflow.
+
+    """
+    try:
+        submission = submissions_api.get_submission_and_student(submission_uuid)
+        workflow, __ = StaffWorkflow.objects.get_or_create(
+            course_id=submission['student_item']['course_id'],
+            item_id=submission['student_item']['item_id'],
+            submission_uuid=submission_uuid
+        )
+        workflow.save()
```

Comment:
Fun fact I've picked up from heavily debugging SQL behind django models - `get_or_create(..)` will include a call to `save(force_insert=True)` if creating, so this `save()` is unneeded.